Tasks/JenkinsQueueJobV2/Tests/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vsts-tasks-jenkinsqueuejob",
   "devDependencies": {
     "@types/mocha": "^5.2.0",
-    "vsts-task-lib": "0.9.20"
+    "vsts-task-lib": "2.4.0"
```

Comment:
Why is this a dev dependency? I think it just happens to work since the way tasks are packaged doesn't exclude dev dependencies at run time.